src/pages/Reports/KindReport.js

```diff
@@ -35,8 +38,18 @@ function KindReport({
     setService(value);
   }
 
+  function onStatusChange(value) {
+    setStatus(value)
+  }
+
+  function onResultClose(id){
+    const url = `/update/${id}`;
+    updateStatus(url, {status: 'closed'});
+    handleSearch();
```

Comment:
for now reloading search is fine, but I was thinking of just updating the row value in state on success result. we can do that in next iteration.